conf/app.routes

```diff
@@ -1,17 +1,18 @@
 # microservice specific routes
 
-
 # subscription field definitions endpoints routes
+PUT           /definition/context/:apiContext/version/:apiVersion                      @uk.gov.hmrc.apisubscriptionfields.controller.FieldsDefinitionController.upsertFieldsDefinition(apiContext: String, apiVersion: String)
 
-# TODO
+GET           /definition/context/:apiContext/version/:apiVersion                      @uk.gov.hmrc.apisubscriptionfields.controller.FieldsDefinitionController.getFieldsDefinition(apiContext: String, apiVersion: String)
 
 
 # subscription fields endpoints routes
+PUT           /field/application/:appId/context/:apiContext/version/:apiVersion        @uk.gov.hmrc.apisubscriptionfields.controller.ApiSubscriptionFieldsController.upsertSubscriptionFields(appId: String, apiContext: String, apiVersion: String)
 
-PUT         /field/application/:appId/context/:apiContext/version/:apiVersion        @uk.gov.hmrc.apisubscriptionfields.controller.ApiSubscriptionFieldsController.upsertSubscriptionFields(appId: String, apiContext: String, apiVersion: String)
+GET           /field/application/:appId/context/:apiContext/version/:apiVersion        @uk.gov.hmrc.apisubscriptionfields.controller.ApiSubscriptionFieldsController.getSubscriptionFields(appId: String, apiContext: String, apiVersion: String)
 
-GET         /field/application/:appId/context/:apiContext/version/:apiVersion        @uk.gov.hmrc.apisubscriptionfields.controller.ApiSubscriptionFieldsController.getSubscriptionFields(appId: String, apiContext: String, apiVersion: String)
+GET           /field/:fieldsId                                                         @uk.gov.hmrc.apisubscriptionfields.controller.ApiSubscriptionFieldsController.getSubscriptionFieldsByFieldsId(fieldsId: java.util.UUID)
 
-GET         /field/:fieldsId                                                         @uk.gov.hmrc.apisubscriptionfields.controller.ApiSubscriptionFieldsController.getSubscriptionFieldsByFieldsId(fieldsId: java.util.UUID)
+GET           /field/application/:appId                                                @uk.gov.hmrc.apisubscriptionfields.controller.ApiSubscriptionFieldsController.getBulkSubscriptionFieldsByApplicationId(appId: String)
 
```

Comment:
Could you please add a new route?
```
GET     /field/
```

This would need to return a JSON with all subscription fields.

I know we do not use it anywhere in the code, but this would be very useful in case of investigations in production for example, where we do not have access to mongoDb.